screens/HomeAuth.js

```diff
@@ -119,9 +134,12 @@ class HomeAuth extends Component {
       }
       this.props.navigation.navigate('menteeListView', { headerTitle });
     } else {
-      const token = await AsyncStorage.getItem('fb_token');
-      if (token) {
-        const success = await this.getHeyMentorToken(token, 'facebook');
+      const fbToken = await AsyncStorage.getItem('fb_token');
+      const gToken = await AsyncStorage.getItem('g_token');
+      if (fbToken || gToken) {
+        const token = fbToken || gToken;
```

Comment:
Can we pop this out and use it for the if clause to prevent having the check twice?

Reply:
makes sense, fixed